src/rules/font-size-is-readable/index.js

```diff
@@ -7,23 +7,23 @@ export const messages = utils.ruleMessages(ruleName, {
   expected: (selector) => `Expected a larger font-size in ${selector}`,
 });
 
-const THRESHOLD_IN_PX = 15;
-
 const pxToPt = (v) => 0.75 * v;
 
-const checkInPx = (value) =>
+const checkInPx = (value, THRESHOLD_IN_PX) =>
   value.toLowerCase().endsWith('px') && parseFloat(value) < THRESHOLD_IN_PX;
-const checkInPt = (value) =>
+const checkInPt = (value, THRESHOLD_IN_PX) =>
   value.toLowerCase().endsWith('pt') && parseFloat(value) < pxToPt(THRESHOLD_IN_PX);
 
-export default function (actual) {
+export default function (actual, options) {
   return (root, result) => {
     const validOptions = utils.validateOptions(result, ruleName, { actual });
 
     if (!validOptions || !actual) {
       return;
     }
 
+    const THRESHOLD_IN_PX = (options && options.thresholdInPixels) || 15;
```

Comment:
@jsnmrs do you think we should be more opinionated and set the default size?
```suggestion
    const THRESHOLD_IN_PX = (options && options.thresholdInPixels) || 16;
```

Reply:
@katydecorah This is tricky because there isn't a minimum font size WCAG guideline. In theory, if the page is properly zoomable (WCAG 1.4.4), a user could adjust font size themselves.

If we do this, we'll have to back it up in the readme. I'd also like to see about catching more edge cases, like size keywords (`font-size: x-small`).

Making the threshold configuration is awesome! Let's move forward with that, but pause on setting a default until we provide rationale.